keadm/cmd/keadm/app/cmd/common/constant.go

```diff
@@ -65,6 +77,24 @@ const (
 	// CertPort is the port where to apply for the edge certificate
 	CertPort = "certport"
 
+	// QuicServer sets the port where to apply for the edge quic
+	QuicPort = "quicport"
+
+	// Used to distinguish download links  for KubeEdge EdgeCore and Services
+	DownloadRegion = "download-region"
+
+	// Tunnel sets the port where to apply for the edge stream
+	TunnelPort = "tunnelport"
+
+	// Custom download url for KubeEdge tar file and checksum file
+	KubeEdgeDownloadURL = "kubeedge-download-url"
+
+	// Custom download url for service file
+	ServiceFileURLFormat = "service-file-url-format"
+
+	// Env file contains extra flags for building edgecore setup configuration
+	KeadmJoinFlagsPath = "/etc/kubeedge/keadm-join-flags.env"
```

Comment:
what's the format of the contents of this document? just like this? 😂 
```
a=b
c=d
```

Reply:
If you have other good ideas, we can make it better :)